package.json

```diff
@@ -74,6 +74,7 @@
     "typescript": "^5.3.3"
   },
   "dependencies": {
+    "ollama": "^0.5.0",
```

Comment:
Note I'm actually not using it, had too many issues with it so went to `fetch()` instead.